src/run.rs

```diff
@@ -46,6 +47,18 @@ pub fn generate_tmp_project(
             command.status().context("Could not start git")?.success(),
             "Cannot create working tree"
         );
+    } else if let Some(url) = git {
+        let mut command = process::Command::new("git");
+        command
+            .arg("clone")
+            .arg("--quiet")
+            .arg(url)
+            .arg(&tmp_dir.as_ref());
```

Comment:
`--quiet` is definitely not a good thing. You want to see logs in a terminal.

Because you clone temporarily, I would argue that `--depth 1` is important. I would make that the default but it could also have a parameter in the config to disable it.